test/badges.test.ts

```diff
@@ -18,6 +18,7 @@ const errMintFailed = 'mintAuthorizedBadge: badge minting failed'
 const errSpecNotRegistered = 'mintAuthorizedBadge: spec is not registered'
 const errSpecAlreadyRegistered = 'createSpecAsRaftOwner: spec already registered'
 const errNotRaftOwner = 'createSpecAsRaftOwner: unauthorized'
+const errInvalidSig = '_safeCheckAgreement: invalid signature'
```

Comment:
`errMintFailed` and `errSpecNotRegistered`

I think we can remove these 2 as they'd be unused right?

Reply:
we use `errSpecNotRegistered` in the final test

Reply:
ah right. That needs to be get updated.. the prefix needs to be `_mint`